contracted-contributor.md

```diff
@@ -0,0 +1,139 @@
+abandons the initiative altogether.
+
+## Forces
+
+- Managers of business units are held accountable for their results. Letting
+  their staff participate in InnerSource activities not contributing to his
+  organization would effectively reduce the capacity of his unit. This will
+  likely make it harder for the managers to reach or exceed their goals.
+- The performance review of all employees is based on what their line managers
+  think they should do. Line managers (and HR) need to consider InnerSource
+  activities as career relevant or else, the performance of employees
+  participating in InnerSource as perceived by their line managers will suffer.
+- The more time an associate spends on contributions to an InnerSource project
+  which does not benefit his day-to-day work, the more will the workload for
+  his teammates in his business unit increase.
+- The individual contributor would like to participate to enhance his
+  professional network within the company and gain knowledge and experience
+  with both the InnerSource method and the technical area he makes a
+  contribution to.
+- The less executive air cover a line manager perceives he has, the less likely
+  is he or she to have his or her staff participate in InnerSource activities
+  which contribute to another business unit.
+- The less transparency and control a line manager has of work done by one of
+  subordinates InnerSource, the less likely is he or she to allow him or her to
+  contribute.
+- The less formal work in InnerSource is managed and organised, the less likely
+  a line manager accustomed to formal processes is to sign off on one of his
+  employees contributing to InnerSource.
+
+## Solution
+
+- Set up a governance office (_tbd. pattern link_) with central funding and
+  budgetary independence from business units.
+- The governance office sets up formal contracting between associate and his
+  line management stating the ground rules for the associate participating in
+  the InnerSource project. The contracting specifies a maximum percentage of
+  the associates work time in InnerSource. The contracting also makes it clear
+  that business usually takes precendence over work in InnerSource and that it
+  is not required to work in InnerSource for the maximum percentage specified.
+  The contracting is signed by the governance office, the community leader and
+  the contributors line manager.
+- The governance office refunds the business unit of the contributor for the
+  time he participates in the InnerSource project with central funding. While
+  that does not solve the business units problems related to meeting project
+  deadlines and capacity for most business units, it does send a clear message
+  to line managers that InnerSource has executive air cover and is something
+  they expect from their organisation.
+- The governance office mediates between the associate and his line management
```

Comment:
Add caveat about this (not very effective measure) into _Resulting Context_ section (line 104)